bindings/node.js/package.json

```diff
@@ -2,26 +2,26 @@
   "name": "c-kzg",
   "version": "1.1.2",
   "description": "NodeJS bindings for C-KZG",
-  "author": "Dan Coffman",
+  "contributors": [
+    "Dan Coffman",
+    "Matthew Keil"
```

Comment:
This contributor list is fine with me :+1: You should add email's tho, dan's `dgcoffman@gmail.com` ([source](https://github.com/dgcoffman)) in this format
```
"Barney Rubble <b@rubble.com>"
```
([ref](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#people-fields-author-contributors))